### PR TITLE
Camunda 8 Run authentication warning

### PR DESCRIPTION
**Pull request by mesellings**
## Description

This PR provides further temporary fixes to help resolve the Camunda 8 Run/REST API authentication issue.

- Changed the warning to a caution, and reworded to remove limitation language and other components.
- Additionally, fixed link format and typo

This does not address further changes required for this authentication issue, as detailed in the relevant Slack channel, namely:

> To access the REST API (v1 or v2) you need to make use of the Cookie auth setup by the webapps and we need to document that too.

Once that documentation is complete, we should revisit these pages and change the caution link to point to the new auth documentation.

## When should this change go live?

- [X] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [X] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [X] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [X] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [X] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

**Review by megglos — APPROVED** [review]
LGTM! Thanks @mesellings for taking action here right away 🚀

File: docs/guides/getting-started-java-spring.md [hunk 1]
```diff
@@ -133,7 +133,7 @@ See our documentation on [adding the Spring Zeebe SDK to your project](/apis-too
 </repositories>
 ```
 
-2. Add the following dependency your `pom.xml` file, as a child of the `<dependencies>` element:
+2. Add the following dependency to your `pom.xml` file, as a child of the `<dependencies>` element:
 
 :::warning
```
Comment (akeller) [inline docs/guides/getting-started-java-spring.md:138]: This warning is still here and should likely be softened with updates to point at the patch. Let's soften the warning so we can get this merged and follow up when the patch is out.

File: versioned_docs/version-8.6/guides/getting-started-java-spring.md [hunk 2]
```diff
@@ -133,7 +133,7 @@ See our documentation on [adding the Spring Zeebe SDK to your project](/apis-too
 </repositories>
 ```
 
-2. Add the following dependency your `pom.xml` file, as a child of the `<dependencies>` element:
+2. Add the following dependency to your `pom.xml` file, as a child of the `<dependencies>` element:
 
 :::warning
```
Comment (akeller) [inline versioned_docs/version-8.6/guides/getting-started-java-spring.md:138]: Same note here

File: docs/guides/getting-started-java-spring.md (hunk 1 above)
Comment (mesellings) [reply docs/guides/getting-started-java-spring.md:138]: Do you just mean warning > caution, or also change the language?

e.g. change to:

:::caution
Due to a bug in 8.6, use the Spring Zeebe SDK version `8.5.0`.
:::

Or to:

:::caution
Due to an issue in 8.6, use the Spring Zeebe SDK version `8.5.0`. This issue will be resolved in a future patch release.
:::

Comment (akeller) [inline docs/guides/getting-started-java-spring.md:138]:
```suggestion
:::caution
```

File: versioned_docs/version-8.6/guides/getting-started-java-spring.md (hunk 2 above)
Comment (akeller) [reply versioned_docs/version-8.6/guides/getting-started-java-spring.md:138]:
```suggestion
:::caution
```

File: docs/guides/getting-started-java-spring.md (hunk 1 above)
Comment (akeller) [reply docs/guides/getting-started-java-spring.md]: Added a commit - your first option is what I went with.

File: docs/guides/getting-started-java-spring.md [hunk 3]
```diff
@@ -133,9 +133,9 @@ See our documentation on [adding the Spring Zeebe SDK to your project](/apis-too
 </repositories>
 ```
 
-2. Add the following dependency your `pom.xml` file, as a child of the `<dependencies>` element:
+2. Add the following dependency to your `pom.xml` file, as a child of the `<dependencies>` element:
 
-:::warning
+:::caution
```
Comment (romansmirnov) [inline docs/guides/getting-started-java-spring.md:138]: IMO, this caution/warning should be removed in favor of https://github.com/camunda/camunda-8-get-started-spring/pull/9/files - In a nutshell, the `application.properties` must be adjusted to make it work.

File: versioned_docs/version-8.6/guides/getting-started-java-spring.md [hunk 4]
```diff
@@ -133,9 +133,9 @@ See our documentation on [adding the Spring Zeebe SDK to your project](/apis-too
 </repositories>
 ```
 
-2. Add the following dependency your `pom.xml` file, as a child of the `<dependencies>` element:
+2. Add the following dependency to your `pom.xml` file, as a child of the `<dependencies>` element:
 
-:::warning
+:::caution
```
Comment (romansmirnov) [inline versioned_docs/version-8.6/guides/getting-started-java-spring.md:138]: The same as above.

File: docs/guides/getting-started-java-spring.md [hunk 5]
```diff
@@ -133,9 +133,9 @@ See our documentation on [adding the Spring Zeebe SDK to your project](/apis-too
...
 Due to a bug in 8.6, use the Spring Zeebe SDK version `8.5.0`.
```
Comment (conceptualshark) [inline docs/guides/getting-started-java-spring.md:139]:
```suggestion
The Spring Zeebe SDK is currently pinned to version `8.5.0`.
```

^ If we'd like to remove the bug language, we can just note that it's currently set at 8.5 (and therefore required).

File: versioned_docs/version-8.6/self-managed/setup/deploy/local/c8run.md [hunk 6]
```diff
@@ -25,8 +25,8 @@ Camunda 8 Run includes the following:
 
 ## Prerequisites
 
-:::warning
-Due to a bug in 8.6, cookie authentication is required to use the Tasklist, Operate, and Zeebe APIs included in Camunda 8 Run. For an example of authenticating via cookie, see the [Operate API documentation](/apis-tools/operate-api//authentication.md#authentication-via-cookie-self-managed-only).
+:::caution
```
Comment (romansmirnov) [inline versioned_docs/version-8.6/self-managed/setup/deploy/local/c8run.md:28]: Being honest, to me this does not make sense. This installation guide does not reference the Camunda REST API at all. The page still focuses on the old architecture by talking about Zeebe configuration, gRPC endpoints, etc.

File: versioned_docs/version-8.6/guides/react-components/install-c8run.md [hunk 7]
```diff
@@ -1,7 +1,7 @@
 ### Prerequisites
 
-:::warning
-Due to a bug in 8.6, cookie authentication is required to use the Tasklist, Operate, and Zeebe APIs included in Camunda 8 Run. For an example of authenticating via cookie, see the [Operate API documentation](/apis-tools/operate-api//authentication.md#authentication-via-cookie-self-managed-only).
+:::caution
```
Comment (romansmirnov) [inline versioned_docs/version-8.6/guides/react-components/install-c8run.md:3]: I think the Camunda REST API is not referenced or promoted in the guides, so why is this "caution" necessary?

**Review by romansmirnov — COMMENTED** [review]
I shared my feedback on Slack already, so I disagree with having such a caution so prominent because it behaves as expected and the Camunda REST API is not really mentioned or used in the guides.

**Comment by akeller** [comment]
> I shared my feedback on Slack already, so I disagree with having such a caution so prominent because it behaves as expected and the Camunda REST API is not really mentioned or used in the guides.

Thanks @romansmirnov. We also have instructions to work with C8Run that may be for non-getting started users. We've softened this quite a bit and removed the admonition so people are proactively informed.

**Review by akeller — APPROVED** [review]
Based on the conversations in the Zoom, this looks like it satisfies feedback in a way that centers the developer experience without being too shouty.

**Pull request merged**